admin/sales_analytics/keen.py

```diff
@@ -0,0 +1,25 @@
+from admin.base.settings import KEEN_PROJECT_ID, KEEN_READ_KEY, KEEN_WRITE_KEY
+
+KEEN_CREDENTIALS = {
+    'keen_ready': False
+}
+
+if KEEN_CREDENTIALS['keen_ready']:
+    KEEN_CREDENTIALS.update({
+        'keen_project_id': KEEN_PROJECT_ID,
+        'keen_read_key': KEEN_READ_KEY,
+        'keen_write_key': KEEN_WRITE_KEY
+    })
+
+
+# TODO: when Keen is ready, use of settings instead
```

Comment:
I planned to wait until Keen becomes ready in OSF. But we can definitely change to the commented code below to use OSF settings now.